desktop/README.md

```diff
@@ -0,0 +1,3 @@
+# RENKU desktop
+
+This project template showcases the latest VNC (virtual desktop) that RENKU comes with.
```

Comment:
This is going to be the README of the actual project - probably want to add some different information here?